dataland-frontend/tests/e2e/specs/eu-taxonomy-financials/AddRemoveKpis.ts

```diff
@@ -0,0 +1,113 @@
+
+describeIf(
+  "As a user, I expect to be able to add and remove Eligible KPIs and send the form successfully",
+  {
+    executionEnvironments: ["developmentLocal", "ci", "developmentCd"],
+    dataEnvironments: ["fakeFixtures"],
+  },
+  function () {
+    beforeEach(() => {
+      cy.ensureLoggedIn(uploader_name, uploader_pw);
+    });
+
+    let testData: FixtureData<EuTaxonomyDataForFinancials>;
+
+    before(function () {
+      cy.fixture("CompanyInformationWithEuTaxonomyDataForFinancialsPreparedFixtures").then(function (jsonContent) {
+        const preparedFixtures = jsonContent as Array<FixtureData<EuTaxonomyDataForFinancials>>;
+        testData = getPreparedFixture("company-for-all-types", preparedFixtures);
+      });
+    });
+
+    it("Check", () => {
+      testData.companyInformation.companyName = "company-for-all-types";
+      companyViaApiAndEuTaxonomyDataForFinancialsViaForm(
+        testData.companyInformation,
+        testData.t,
+        () => undefined,
+        fillAndValidateEuTaxonomyForFinancialsUploadForm,
+        () => undefined,
+        () => undefined,
+        () => undefined
+      );
+    });
+  }
+);
+
+/**
+ * Fills the eutaxonomy-financials upload form with the given dataset
+ * @param data the data to fill the form with
+ */
+function fillAndValidateEuTaxonomyForFinancialsUploadForm(data: EuTaxonomyDataForFinancials): void {
+  dateFormElement.selectDayOfNextMonth("fiscalYearEnd", 12);
+  dateFormElement.validateDay("fiscalYearEnd", 12);
+
+  // Required data entry
+  if (data.reportingObligation !== undefined) {
+    cy.get(`input[name="reportingObligation"][value=${data.reportingObligation.toString()}]`).check();
+  }
+
+  cy.get(
+    `input[name="fiscalYearDeviation"][value=${
+      data.fiscalYearDeviation ? data.fiscalYearDeviation.toString() : "Deviation"
+    }]`
+  ).check();
+
+  cy.get('input[name="numberOfEmployees"]').type(
+    `${data.numberOfEmployees ? data.numberOfEmployees.toString() : "13"}`
+  );
+
+  cy.get('[data-test="assuranceSection"] select[name="assurance"]').select(2);
+  cy.get('[data-test="assuranceSection"] input[name="provider"]').type("Assurance Provider", { force: true });
+  cy.get('[data-test="assuranceSection"] select[name="report"]').select(1);
+
+  cy.get('[data-test="MultiSelectfinancialServicesTypes"]')
+    .click()
+    .get("div.p-multiselect-panel")
+    .find("li.p-multiselect-item")
+    .each(($el) => {
+      cy.wrap($el).click({ force: true });
+    });
+
+  cy.get('[data-test="addKpisButton"]').click({ force: true });
+
+  cy.get('[data-test="removeSectionButton"]').each(($el, index) => {
+    if (index > 0) {
+      cy.wrap($el).click({ force: true });
+    }
+  });
+
+  cy.get('[data-test="addKpisButton"]').click({ force: true });
+  cy.get('button[data-test="removeSectionButton"]').should("exist").should("have.class", "ml-auto");
+
+  cy.get('[data-test="dataPointToggle"]')
```

Comment:
Why do you test the toggling of a datapoint here?

Reply:
Removed. Done.